BootloaderCorePkg/Stage2/Stage2Support.c

```diff
@@ -390,6 +390,9 @@ BuildBaseInfoHob (
   if (LoaderFspInfo != NULL) {
     LoaderFspInfo->FspsBase   = PCD_GET32_WITH_ADJUST (PcdFSPSBase);
     LoaderFspInfo->FspHobList = LdrGlobal->FspHobList;
+    LoaderFspInfo->VerifiedBoot = FeaturePcdGet (PcdVerifiedBootEnabled);
+    LoaderFspInfo->MeasuredBoot = FeaturePcdGet (PcdMeasuredBootEnabled);
```

Comment:
we could not just use these

















Even we build Verified boot and Measured boot (FeaturePcds are set to true in SBL), but the boot guard profile could disable them. should we use LOADER_PLATFORM_INFO->SecurityFlags instead?

Reply:
Sure, I would use interface LOADER_PLATFORM_INFO->SecurityFlags to update the security features.